src/tcp_cubic_search.c

```diff
@@ -433,6 +463,11 @@ static void bictcp_state(struct sock *sk, u8 new_state)
 {
 	if (new_state == TCP_CA_Loss) {
 		bictcp_reset(inet_csk_ca(sk));
+
```

Comment:
since we are using union here.  We may need to check the hystart mode to do reset. 
otherwise, it may overwriting each other.